admin-node-setup.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/sh
 # this script WILL BE RUN ON EVERY REBOOT
 
+# Clear the transactional update grains when booting up
+if [ -f /etc/salt/grains ]; then
+    sed -i -e 's|tx_update_reboot_needed:.*|tx_update_reboot_needed: false|g' /etc/salt/grains
```

Comment:
A `sed -i "/tx_update_reboot_needed/d" /etc/salt/grains` would just remove the line